### DISCUSSION TITLE
Suddenly my SAM and MOOSE system does not work

### DISCUSSION THREAD

**Discussion by Terryxixi2014** [idaholab/moose#16435] [Q&A General] [answered]
I got an error: error while loading shared libraries: libmoose-opt.so.0: cannot open shared object file: no such file or directory

suddenly, I cannot start any new calculations. The above error is there. Any idea?

**Comment by GiudGiud** [comment, marked as answer]
Hi Terry,

Did you update SAM before this?
Are you loading the moose environment? (if you are using the conda moose install)
I would try re-building moose. 

Guillaume
